src/build.rs

```diff
@@ -11,10 +11,30 @@ pub fn rustup_add_wasm_target() -> Result<(), Error> {
         emoji::TARGET
     );
     let pb = PBAR.message(&step);
+    let nightly_check = Command::new("rustc").arg("+nightly").arg("-V").output()?;
```

Comment:
This whole part should be it's own section/fn as there's no sub bar really. It should just be part of the process.

Reply:
Just pushed an update with this change